build.analyze.sh

```diff
@@ -74,9 +74,14 @@ fi
 mkdir -p $BUILD_DIR
 mkdir -p $REPORT_DIR
 rm -Rf $BUILD_DIR/*
+rm -Rf $REPORT_DIR/*
 (cd $BUILD_DIR && $BUILD_WRAPPER $ANALYZE_OPTS \
                   cmake -DCMAKE_BUILD_TYPE=Release \
                         -DENVIRONMENT_TYPE=default \
+                        \
+                        -DCMAKE_C_COMPILER=clang \
+                        -DCMAKE_CXX_COMPILER=clang++ \
+                        \
```

Comment:
That is not necessary, but propably this does not work with `gcc` due to the precompiled headers (which are only enabled in gcc builds).  Better disable `ENABLE_COMPTIME_RED`in this builds.